get-started/part2.md

```diff
@@ -138,6 +138,12 @@ At this point, you've successfully built an image, performed a simple containeri
 
 [On to Part 3 >>](part3.md){: class="button outline-btn" style="margin-bottom: 30px; margin-right: 100%"}
 
+## Deploying to the cloud
+
+If you want to go ahead and get your containers running in either Azure or AWS then check out our blog posts on getting started with Cloud deployments and Docker
+* [Deploying with Docker and AWS](https://www.docker.com/blog/from-docker-straight-to-aws/)
```

Comment:
Can we point to the docs instead of the blogs? The detailed steps are documented at 

- https://docs.docker.com/engine/context/aci-integration/
- https://docs.docker.com/engine/context/ecs-integration/

Reply:
Changed to these links